src/hera.cpp

```diff
@@ -371,12 +372,18 @@ evmc_result hera_execute(
   } catch (InternalErrorException const& e) {
     ret.status_code = EVMC_INTERNAL_ERROR;
     HERA_DEBUG << "InternalError: " << e.what() << "\n";
+    if (hera->fuzzing)
+      abort();
   } catch (exception const& e) {
     ret.status_code = EVMC_INTERNAL_ERROR;
     HERA_DEBUG << "Unknown exception: " << e.what() << "\n";
+    if (hera->fuzzing)
+      abort();
   } catch (...) {
     ret.status_code = EVMC_INTERNAL_ERROR;
     HERA_DEBUG << "Totally unknown exception\n";
+    if (hera->fuzzing)
+      abort();
   }
 
```

Comment:
```suggestion
if (ret.status_code == EVMC_INTERNAL_ERROR)
  abort();
```

Reply:
True. Though probably I prefer this way because the branch is only triggered in case of an error.